infra/dev/docker-compose.yml

```diff
@@ -0,0 +1,77 @@
+version: "3.7"
+services:
+  nginx:
+    image: nginx:stable
+    container_name: dev_nginx
+    ports:
+      - 80:80
+    volumes:
+      - ./nginx/nginx.conf:/etc/nginx/nginx.conf
+    depends_on:
+      - laravel
+    networks:
+      - paul_net
+
+  # Here we delegate the dependency installation to composer container
+  composer:
```

Comment:
Composer is php dependency manager hence it should be in the php container (or laravel since you named it that way)

Reply:
I separate it, since I found out there is a composer container already available (avoid the installation just for one command)

Reply:
Can't run `composer install` on Laravel as we map the app source in that container, and those files are only present during runtime and not buildtime.